app/models/message.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: messages
+#
+#  id              :integer          not null, primary key
+#  conversation_id :integer
+#  sender_id       :integer          not null
+#  content         :string           not null
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+# Indexes
+#
+#  index_messages_on_conversation_id  (conversation_id)
+#  index_messages_on_sender_id        (sender_id)
+#
+
+class Message < ApplicationRecord
+  belongs_to :conversation
+  belongs_to :sender, class_name: :User
+
+  validates_presence_of :content
```

Comment:
Use the new syntax as this one is deprecated

Reply:
validates :content, presence: true